src/renderers/dom/stack/client/ReactDOMComponent.js

```diff
@@ -552,7 +552,7 @@ ReactDOMComponent.Mixin = {
           div.innerHTML = `<${type}></${type}>`;
           el = div.removeChild(div.firstChild);
         } else if (props.is) {
-          el = ownerDocument.createElement(type, props.is);
+          el = ownerDocument.createElement(type, {is: props.is});
```

Comment:
See https://dom.spec.whatwg.org/#dom-document-createelement.

The `is` prop indicates that the react element should be a customized builtin element.